### DISCUSSION TITLE
create customers without using projects

### DISCUSSION THREAD

**Discussion by mastertIT** [kimai/kimai#4376] [General] [answered]
trying to setup kimai.
service based business.
we have fixed customers that we don't do project based time on.

we setup activities which are basic service calls or installs.

currently there is no way to create only customers.

how would we use it so a tech only needs to choose the customer and the type of work being done?

**Comment by kevinpapst** [comment]
create one default "support" project for each customer.
kimai will choose the entry automatically if there is only one.
you could even hide the project field via custom css if you want to.

**Comment by mastertIT** [comment]
wouldn't a project need to be created 1 by 1 for every customer this way?

**Comment by mastertIT** [comment]
is there a way to avoid this redundant task?
_project billing bool = true (default) > stay as is
project billing bool = false > projects use the customer tables and fields._

the issue is we tried using just projects as the dropdown but with no way to add addresses and other columns to a project type it was severely limited

**Reply by jassonmc** [reply]
Well, you could add the plugin "custom fields" and extend your project with address fields.
You then just setup one default customer that you are using for all those projects.

You approach your issue then from the other way; one customer, many projects.

**Reply by mastertIT** [reply, marked as answer]
Yes that's the direction we're going, could not find a way to deal with address's so we just have each 'project' titled 'name - city - address'

It works out pretty well.

we're working on scripting it so each row is directly inserted into the project db table
